src/main/scala/dpla/ingestion3/mappers/utils/Extractor.scala

```diff
@@ -91,6 +91,17 @@ trait XmlExtractor extends Extractor[NodeSeq] {
     getByAttribute(e.asInstanceOf[Elem], att, value)
   }
 
+  def filterAttributeListOptions(node: Node, att: String, values: Seq[String]): Boolean =
```

Comment:
Can you write a quick definition of these methods? E.g. given X what do they return?

Reply:
Yes, thank you for reminding me!